README.md

```diff
@@ -1,11 +1,160 @@
+[![Hex pm](https://img.shields.io/hexpm/v/kaur.svg)](https://hex.pm/packages/kaur)
+[![Build
+Status](https://travis-ci.org/fewlinesco/kaur.svg?branch=master)](https://travis-ci.org/fewlinesco/kaur)
+[![Inline docs](http://inch-ci.org/github/fewlinesco/kaur.svg)](http://inch-ci.org/github/fewlinesco/kaur)
+
 # Kaur
```

Comment:
How about adding a small note about the pronunciation?

The way they do in dictionaries etc...

```
> Pronounced |kɔː|
```